nicotine.spec

```diff
@@ -0,0 +1,80 @@
+# -*- mode: python ; coding: utf-8 -*-
```

Comment:
Hey @mathiascode. Your PR is generally fine, with the exception of this file. I'd suggest moving RPM related stuff into a separate PR, just in case someone has issues later. It helps keep things organized. Otherwise everything looks fine.

Reply:
Will do.

Reply:
Wait, this spec file isn't for generating an RPM.